tests-dev/machine_config/machine_hera.config

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eux
+
+module load rocoto
+ROCOTORUN=/apps/rocoto/1.3.7/bin/rocotorun
+ROCOTOSTAT=/apps/rocoto/1.3.7/bin/rocotostat
+ROCOTOCOMPLETE=/apps/rocoto/1.3.7/bin/rocotocomplete
```

Comment:
Just curious why you leaned towards hardcoding these variables instead of using `which` ?